eng/testing/performance/ios_scenarios.proj

```diff
@@ -26,19 +26,25 @@
   <ItemGroup>
     <HelixWorkItem Include="SOD - iOS HelloWorld .app Size">
         <PayloadDirectory>$(WorkItemDirectory)</PayloadDirectory>
-        <PreCommands>cd $(ScenarioDirectory)helloios;xcopy %HELIX_CORRELATION_PAYLOAD%\iosHelloWorld\$(LlvmPath) .\app\/e;$(Python) pre.py</PreCommands>
+        <PreCommands>cd $(ScenarioDirectory)helloios;xcopy %HELIX_CORRELATION_PAYLOAD%\iosHelloWorld\$(LlvmPath) .\app/e/i;$(Python) pre.py --name app</PreCommands>
```

Comment:
I am not sure I understand the `--name app` part. Is there a file that is created that is just called app?

Reply:
Yes, app is the folder that the llvm is copied into in the xcopy step. Ultimately, the addition of the name app part is to support the new pre.py that doesn't assume app is the folder everything is copied into.